var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -99,7 +99,7 @@ def set_variables(self, env):
         env.set('HIP_CLANG_PATH', rocm_prefixes['llvm-amdgpu'].bin)
         env.set('HSA_PATH', rocm_prefixes['hsa-rocr-dev'])
         env.set('ROCMINFO_PATH', rocm_prefixes['rocminfo'])
-        env.set('DEVICE_LIB_PATH', rocm_prefixes['device_lib_path'])
+        env.set('DEVICE_LIB_PATH', '/opt/rocm/lib')
```

Comment:
This was not supposed to be pushed I suppose?

Reply:
This got it "working", but yes it is ugly. I will adjust rocm_prefixes above for a cleaner fix in a few moments, I pushed things a bit early yes.

Reply:
I am also not entirely sure why there is both rocm-device-libs and device_lib_path in the dictionary, seems redundant. I'll delete one.

Reply:
```suggestion
        rocm_prefixes['device_lib_path']
```
should do, right?

Reply:
Sorry, I made this more confusing than necessary, but i introduced `device_lib_path` exactly because the spack build requires some odd path into the libs folder. I probably forgot to set the `device_lib_path` in external build to `[rocm_path].lib`

Reply:
if `rocm-device-libs` is not used, feel free to drop it :)

Reply:
I decided to include both so the dictionary would have the same number of entries for both external and internal. But for my purposes they will just be set to the same thing